consul/server.go

```diff
@@ -371,6 +374,7 @@ func (s *Server) setupRaft() error {
 
 	// Make sure we set the LogOutput.
 	s.config.RaftConfig.LogOutput = s.config.LogOutput
+	s.config.RaftConfig.SnapshotInterval = raftSnapshotInterval
```

Comment:
Since this is a constant and not a late-bound thing like the logger, this should probably go into `DefaultConfig()` over config.go.